google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -76,21 +76,24 @@ public OSProcessHandler createProcessHandler(
             AppEngineServerModel runConfiguration;
 
             try {
-              // Getting the clone so the debug flags aren't added to the persisted settings.
+              // Getting the clone so the debug com.google.cloud.tools.intellij.flags aren't added
+              // to the persisted settings.
               runConfiguration = (AppEngineServerModel) commonModel.getServerModel().clone();
             } catch (CloneNotSupportedException ee) {
               throw new ExecutionException(ee);
             }
 
             Map<String, String> environment = Maps.newHashMap(configuredEnvironment);
 
-            // IntelliJ appends the JVM flags to the environment variables, keyed by an empty
+            // IntelliJ appends the JVM com.google.cloud.tools.intellij.flags to the environment
```

Comment:
same here

Reply:
yep! fixed.

Reply:
its still showing: com.google.cloud.tools.intellij.flags. Forgot to push maybe?